tests/test_Pretrain.py

```diff
@@ -13,26 +13,6 @@
 }
 
 
-config_json = """
-{
-"lstm0_fw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : false },
-"lstm0_bw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : true },
-
-"lstm1_fw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : false,
-  "from" : ["lstm0_fw", "lstm0_bw"] },
-"lstm1_bw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : true,
-  "from" : ["lstm0_fw", "lstm0_bw"] },
-
-"lstm2_fw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : false,
-  "from" : ["lstm1_fw", "lstm1_bw"] },
-"lstm2_bw" : { "class" : "lstm_opt", "n_out" : 500, "dropout": 0.1, "sampling" : 1, "reverse" : true,
-  "from" : ["lstm1_fw", "lstm1_bw"] },
-
-"output" :   { "class" : "softmax", "loss" : "ce", "from" : ["lstm2_fw", "lstm2_bw"] }
-}
-"""
```

Comment:
You said there was a difference. (Not JSON but sth else?)

Then this should not be removed.